README.md

```diff
@@ -85,7 +85,7 @@ Clone this repository to your desired folder:
 👤 **Emeka Emmanuel**
 
 - GitHub: [@emekaemmanuel](https://github.com/emekaemmanuel)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/emeka-ugboaja-167820226)
+- LinkedIn: [Emeka Ugboaja](https://linkedin.com/in/emeka-ugboaja-167820226)
 
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p> 
```

Comment:
- [ ] Please use semantic tags here, the best one to use is [figure tag], this will improve SEO (https://www.w3schools.com/tags/tag_figure.asp)
![image](https://user-images.githubusercontent.com/74744787/210812554-4e64f3c1-909d-4ba1-a316-ec028de40754.png)

Reply:
✔️